IsBanken.ConsoleApp/Program.cs

```diff
@@ -299,7 +299,7 @@ private static void SearchCustomer()
             Console.WriteLine("* Sök kund *");
             Console.Write("Ange namn eller postort: ");
 
-            var term = Console.ReadLine();
+            var term = Console.ReadLine().ToLower();
```

Comment:
Ta bort toLower() här och flytta in det i customerhandler.CustomerSearchByNameOrCity